azure/Guidelines.md

```diff
@@ -1279,6 +1279,31 @@ In addition to distributed tracing, Azure also uses a set of common correlation
 |x-ms-client-request-id       |Both      |Optional. Caller-specified value identifying the request, in the form of a GUID with no decoration such as curly braces (e.g. `x-ms-client-request-id: 9C4D50EE-2D56-4CD3-8152-34347DC9F2B0`). If the caller provides this header the service **must** include this in their log entries to facilitate correlation of log entries for a single request. Because this header can be client-generated, it should not be assumed to be unique by the service implementation.
 |x-ms-request-id              |Response  |Required. Service generated correlation id identifying the request, in the form of a GUID with no decoration such as curly braces. In contrast to the the `x-ms-client-request-id`, the service **must** ensure that this value is globally unique. Services should log this value with their traces to facilitate correlation of log entries for a single request.
 
+<a name="serversentevents"></a>
+### Server Sent Events (SSE)
+For services that generate partial results over time, it is often beneficial to return the data as soon as it is available to reduce latency. A specific example of this is the generate method for Open AI. One option for this is [Server Sent Events (SSE)](https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events). SSE may be used under the following conditions:
```

Comment:
Is there a description of how SSE works at the HTTP level?  Does it use "chunked responses"?

Reply:
Chunked responses is a Transport-Encoding. SSE is a content encoding. See [this](https://html.spec.whatwg.org/multipage/server-sent-events.html#parsing-an-event-stream) for more info.